<?xml version="1.0"?>
<launch>
  <arg name="model" default="$(find abb_irb14000_support)/urdf/irb14000.xacro" />
  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />

  <arg name="controller_parameters" default="$(find folding_assembly_controller)/input/config_controller.yaml"/>
  <arg name="estimator_parameters" default="$(find folding_assembly_controller)/input/config_estimator.yaml"/>

  <rosparam command="load" file="$(arg controller_parameters)"/>
  <rosparam command="load" file="$(arg estimator_parameters)"/>

  <node name="folding_assembly_node" pkg="folding_assembly_controller" type="folding_assembly_node" output="screen"/>
  <node name="estimator_node" pkg="folding_assembly_controller" type="estimator_node" output="screen"/>
</launch>
